src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costOfDay = 40;
+  const sumOfAllTime = days * costOfDay;
+
+  if (days >= 7) {
+    return (sumOfAllTime - 50);
+  } else if (days < 7 && days >= 3) {
+    return (sumOfAllTime - 20);
+  } else {
+    return sumOfAllTime;
+  }
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 50, 20, 7 and 3 are

Reply:
checklist 3rd item